core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -396,8 +408,11 @@ public double getMaxspeed(EdgeIteratorState edge, int accessType, boolean revers
 
     @Override
     public long flagsDefault(boolean forward, boolean backward) {
-        // just pick car mode to set access values?
-        throw new RuntimeException("do not call flagsDefault");
+//         just pick car mode to set access values?
+//        throw new RuntimeException("do not call flagsDefault");
+    	// TODO This is called on each of the encoders so I had to replace the runtime exception with something,
+    	// but I'm not sure this is correct.
+    	return setAccess(0, forward, backward);
```

Comment:
I think `speedEncoder` was `null`, but I'll check again. I did not want to change the `DataFlagEncoder` a lot, that did not feel completely finished yet and it was not in the focus.